### PR TITLE
Fix/input number suffix support

### PR DESCRIPTION
**Pull request by clayton-duarte**
Fix/input number suffix support

### CODE REVIEW

File: src/components/InputNumber/index.tsx [hunk 1]
```diff
@@ -95,12 +98,28 @@ const InputNumber: FunctionComponent<InputNumberProps &
     setValueAndReturn(controlledValue)
   }, [])
 
+  useEffect(() => {
+    const setCursor = (): void => {
+      if (suffix.length > 0) {
+        const inputElem = document.getElementById(name) as HTMLInputElement
```
Comment (l0rdcafe) [inline src/components/InputNumber/index.tsx:104]: oh no haha

File: src/components/InputNumber/index.tsx [hunk 2]
```diff
@@ -95,12 +98,28 @@ const InputNumber: FunctionComponent<InputNumberProps &
...
+        if (inputElem != null) {
+          const position = value.length - suffix.length
+          inputElem.setSelectionRange(position, position)
```
Comment (l0rdcafe) [inline src/components/InputNumber/index.tsx:107]: question: what does this witchery do?

**Review by l0rdcafe — APPROVED** [review]

File: src/components/InputNumber/index.tsx (hunk 2 above)
Comment (clayton-duarte) [reply src/components/InputNumber/index.tsx:107]: hahah, it moves the caret/cursor (that tick inside the input I don't know the name) to a given position

File: src/components/InputNumber/index.tsx (hunk 1 above)
Comment (clayton-duarte) [reply src/components/InputNumber/index.tsx:104]: hehhehe, I'm planning to refactor it and use Ref later, for know it works fine

**Pull request merged**